styled-components is not installed as a dependency

From version 2.16.0 of the @royalnavy/react-component-library pacakage, styled-components is listed as a peer dependency. However there is no documentation in the version history that says that this package is now also required to be installed by the user in order to use the react-component-library.

Either styled-components should be included as a dependency not a peer dependency or the documentation should be updated to reflect this need.

### Steps to reproduce
1. Include @royalnavy/react-component-library pacakage in your package.json
2. Run npm install
3. Run npm start
4. Notice compilation error 
"./node_modules/@royalnavy/react-component-library/dist/es/components/ContextMenu/ContextMenu.js
Module not found: Can't resolve 'styled-components' in '.../node_modules/@royalnavy/react-component-library/dist/es/components/ContextMenu'

--- a/README.md
+++ b/README.md
@@ -34,6 +34,8 @@ npm install @royalnavy/css-framework @royalnavy/react-component-library
 yarn add @royalnavy/css-framework @royalnavy/react-component-library
 ```
 
+Note: As of `2.16.0` the [`styled-components`](https://github.com/styled-components/styled-components) package is now a required [peerDependency](https://nodejs.org/en/blog/npm/peer-dependencies/).
+
 ### Quick start
 
 Here's a quick example application to get you started:

--- a/packages/docs-site/src/library/pages/get-started/development/installation.md
+++ b/packages/docs-site/src/library/pages/get-started/development/installation.md
@@ -18,6 +18,8 @@ npm install @royalnavy/css-framework @royalnavy/react-component-library
 yarn add @royalnavy/css-framework @royalnavy/react-component-library
 ```
 
+Note: As of `2.16.0` the [`styled-components`](https://github.com/styled-components/styled-components) package is now a required [peerDependency](https://nodejs.org/en/blog/npm/peer-dependencies/).
+
 ### Quick start
 
 Here's a quick example of an application to get you started:

